request.schema.json

```diff
@@ -12,6 +12,105 @@
             "type": "string",
             "pattern": "^data:image/[^;]+;base64,[a-zA-Z0-9+/=]+$"
         },
+        "highChartsData": {
```

Comment:
If this format would potentially be generic enough to be used for data from other charts, suggest naming it more generically, e.g., just "chartData"? @JRegimbal - thoughts?

Reply:
This seems to just be the plain data taken from highcharts. I'd argue we shouldn't give it a generic name since it would conflict with any truly generic input that's accepted later.

Reply:
OK, I am fine with locking this to highcharts, and revisiting after CSUN when we more deeply address a generic format for charts that could be used for multiple chart sources.